apps/rush-lib/src/logic/ProjectChangeAnalyzer.ts

```diff
@@ -29,6 +30,9 @@ export interface IGetChangedProjectsOptions {
   targetBranchName: string;
   terminal: ITerminal;
   shouldFetch?: boolean;
+
+  includeLockfile: boolean;
+  enableFiltering: boolean;
```

Comment:
```suggestion
  /**
   * If set to `true`, consider the project's external dependencies when determining if it has changed.
   */
  includeLockfile: boolean;
  
  /**
   * If set to `true`, ignore files specified in the project's `rush-project.json`'s `incrementalBuildIgnoredGlobs` property.
   */
  enableFiltering: boolean;
```

`includeLockfile` might also make more sense as `includeExternalDependencies` or `includeProjectDependenciesFromLockfile`.